doc/release-notes.md

```diff
@@ -4,3 +4,56 @@ release-notes at release time)
 Notable changes
 ===============
 
+Mnemonic Recovery Phrases
+-------------------------
+
+The zcashd wallet has been modified to support ZIP 339 (derived from BIP 39)
+which describes how to derive the wallet's HD seed from a mnemonic phrase.
+The mnemonic phrase will be generated on load of the wallet, or the first time
+the wallet is unlocked, and is available via the z_exportwallet RPC call. All
+new addresses produced by the wallet are now derived from this seed using the
+HD wallet functionality described in ZIP 32 and ZIP 316. For users upgrading an
+existing Zcashd wallet, it is recommended that the wallet be backed up prior to
+upgrading to the 4.5.2 Zcashd release.
+
+Following the upgrade to 4.5.2, Zcashd will require that the user find their
+new emergency recovery phrase in the output of  the `z_exportwallet` RPC call, 
+and then verify that they have backed up the new recovery phrase by making a
+one-time call to the newly added `walletconfirmbackup` RPC method before the 
+wallet will allow the generation of new addresses. It is recommended that 
+after this upgrade, that funds tied to preexisting addresses be migrated to newly
+generated addresses so that all wallet funds are recoverable using the emergency
+recovery phrase going forward. If you choose not to migrate funds in this fashion,
+you will continue to need to securely back up the entire wallet.dat file to ensure
+that you do not lose access to existing funds; EXISTING FUNDS WILL NOT BE 
+RECOVERABLE USING THE EMERGENCY RECOVERY PHRASE UNLESS THEY HAVE BEEN MOVED TO A
+NEWLY GENERATED ADDRESS FOLLOWING THE 4.5.2 UPGRADE.
+
+New RPC Methods
+---------------
+
+- 'walletconfirmbackup' This newly created API is checks a provided emergency
+  recovery phrase against the wallet's emergency recovery phrase; if the phrases
+  match then it updates the wallet state to allow the generation of new addresses.
+  This backup confirmation workflow can be disabled by starting zcashd with 
+  `-requirewalletbackup=false` but this is not recommended unless you know what
+  you're doing (and have otherwise backed up the wallet's recovery phrase anyway.)
+  If you are using zcash-cli, it is recommended that you only use the `walletconfirmbackup`
+  in a fashion that does not result in the mnemonic phrase that you are confirming
+  backup of being recorded in the shell's history. 
+  See https://unix.stackexchange.com/questions/10922/temporarily-suspend-bash-history-on-a-given-shell
+  for a number of approaches to keeping the phrase out of your shell's history, and
+  be sure to test and verify the approach you choose prior to using it with your real
+  emergency recovery phrase!
```

Comment:
After the above change, reframe this as "do not use this directly from the CLI because (rest of useful content)".